include/wlr/types/wlr_keyboard.h

```diff
@@ -58,7 +58,7 @@ struct wlr_keyboard {
 	xkb_mod_index_t mod_indexes[WLR_MODIFIER_COUNT];
 
 	uint32_t keycodes[WLR_KEYBOARD_KEYS_CAP];
-	size_t num_keycodes;
+	uint32_t num_keycodes;
```

Comment:
This should stay `size_t`

Reply:
Then I should make `button_count` `size_t` to mirror that.